api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/EncounterTranslatorImplTest.java

```diff
@@ -83,7 +83,9 @@
 	
 	private static final String TYPE_CODE = "encounter-type-code";
 	
-	private static final String TYPE_DISPLAY = "encounter-type-display";
+	private static final String TYPE_DISPLAY = "Encounter";
+	
+	private static final String ENCOUNTER_TYPE_CODE = "Encounter";
```

Comment:
```suggestion
	private static final String ENCOUNTER_TYPE_CODE = "encounter";
```

Shouldn't this be `TYPE_CODE` above?